pnpm-lock.yaml

```diff
@@ -7519,6 +7519,16 @@ packages:
       jose: 4.14.4
     dev: true
 
+  /@logto/cloud@0.2.5-9257c0d(zod@3.22.4):
```

Comment:
some package in this repo didn't update `@logto/cloud`

Reply:
Let me check

Reply:
Updated